src/linux/backend-v4l2.cpp

```diff
@@ -2767,6 +2767,15 @@ namespace librealsense
 
         control_range v4l_mipi_device::get_pu_range(rs2_option option) const
         {
+            // Auto controls range is trimed to {0,1} range
+            if(option >= RS2_OPTION_ENABLE_AUTO_EXPOSURE && option <= RS2_OPTION_ENABLE_AUTO_WHITE_BALANCE)
```

Comment:
Do we know what is the actual range sent by FW?
According to that we can decide if it's a FW bug and they should fix or a FW limitation and we will fix here.